### DISCUSSION TITLE
[site] Light mode looks broken

### DISCUSSION THREAD

**Discussion by babakfp** [vercel/turborepo#6852] [Help] [answered]
Hi

https://github.com/vercel/turbo/assets/44144724/8bee1949-3044-4806-9317-4fe84a80e45f

I created this issue here because I didn't know where to open it!

**Comment by anthonyshew** [comment, marked as answer]
Oops, this'll be my mistake!

I've created an issue from this at #6856 if you'd like to follow along there.
